source/iterator/combined.go

```diff
@@ -83,7 +83,7 @@ func (c *Combined) HasNext(ctx context.Context) (bool, error) {
 			return false, fmt.Errorf("snapshot has next: %w", err)
 		}
 
-		if !hasNext && c.isCDCSupported() {
```

Comment:
why we removed it?

Reply:
I think it's because @alyakimenko has removed resources that do not support CDC.

Reply:
Yeah, @voscob, you're right. The connector doesn't support resources that don't have CDC support.

Reply:
Do we add this info to readme? Or it will be other request?

Reply:
Yes, for sure. I will add a matrix of available resources and operations they support when I write documentation.

Reply:
cool , thank you!